connector/connector.py

```diff
@@ -443,10 +443,10 @@ def unwrap_binding(self, binding_id, browse=False):
         else:
             binding = self.model.browse(binding_id)
 
-        openerp_record = getattr(binding, self._openerp_field)
+        record = getattr(binding, self._openerp_field)
```

Comment:
@lasley @lmignon actually I planned to go trough all these `getattr` (there are several of them) and replace them, but as Laurent pointed out there's no user input so I think that we can leave with this for a while :)